src/dynarec/la64/dynarec_la64_660f.c

```diff
@@ -316,6 +316,81 @@ uintptr_t dynarec64_660F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
                     GETEX(q1, 0, 0);
                     VSIGNCOV_W(q0, q1, q0);
                     break;
+                case 0x0B:
+                    INST_NAME("PMULHRSW Gx,Ex");
+                    nextop = F8;
+                    GETGX(q0, 1);
+                    GETEX(q1, 0, 0);
+                    v0 = fpu_get_scratch(dyn);
+                    v1 = fpu_get_scratch(dyn);
+                    VMULWEV_W_H(v0, q0, q1);
+                    VMULWOD_W_H(v1, q0, q1);
+                    VSRAI_W(v0, v0, 14);
+                    VSRAI_W(v1, v1, 14);
+                    VADDI_WU(v0, v0, 1);
+                    VADDI_WU(v1, v1, 1);
+                    VSRANI_H_W(v1, v0, 1);
+                    VSHUF4I_W(v1, v1, 0xd8);
+                    VSHUF4I_H(q0, v1, 0xd8);
+                    break;
+                case 0x1C:
+                    INST_NAME("PABSB Gx,Ex");
+                    nextop = F8;
+                    GETEX(q1, 0, 0);
+                    GETGX_empty(q0);
+                    v0 = fpu_get_scratch(dyn);
+                    VREPLGR2VR_D(v0, xZR);
```

Comment:
Use `VXOR_V(v0, v0, v0);`, which has determined latency and IPC, and it's already used everywhere.